src/Makefile.am

```diff
@@ -512,7 +511,6 @@ libqueso_include_HEADERS += contrib/getpot/getpot.h
 
 nodist_libqueso_include_HEADERS =
 nodist_libqueso_include_HEADERS += $(top_builddir)/config_queso.h
-nodist_libqueso_include_HEADERS += $(top_builddir)/src/core/inc/queso.h
```

Comment:
(Unless, of course, they built and installed `QUESO` from the repository source tree)